src/edit/actions/OpacityAction.js

```diff
@@ -1,9 +1,9 @@
 L.OpacityAction = L.EditAction.extend({
   initialize(map, overlay, options) {
-    var edit = overlay.editing;
-    var mode = edit._mode;
-    var use;
-    var tooltip;
+    let edit = overlay.editing;
```

Comment:
The 'const' keyword should be used here rather than 'let'.

Reply:
@PeculiarE I have just applied the changes, pls do review again thanks